apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -0,0 +1,100 @@
+import base64
+import json
+from pathlib import Path
+
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+def prepare(element_html: str, _: pl.QuestionData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    pl.check_attribs(element, ["name"], ["width", "height", "file", "file_dir"])
+
+
+def render(element_html: str, data: pl.QuestionData) -> str:
+    element = lxml.html.fragment_fromstring(element_html)
+    with open("pl-excalidraw.mustache", "r", encoding="utf-8") as template:
```

Comment:
You should only use the `with open(...) as template` only right before the template is actually used (i.e. indent as little as possible).